crates/next-core/src/next_client_component/with_client_chunks.rs

```diff
@@ -154,20 +168,8 @@ impl ChunkItem for WithClientChunksChunkItem {
     }
 
     #[turbo_tasks::function]
-    async fn references(&self) -> Result<AssetReferencesVc> {
-        let inner = self.inner.await?;
-        Ok(AssetReferencesVc::cell(vec![
-            WithClientChunksAssetReference {
```

Comment:
Was this intended? The reference struct isn't used anywhere else.

Reply:
yeah, it was weird before... Now it's simpler

Reply:
You should remove the `WithClientChunksAssetReference` then.

Reply:
That's still use above